src/components/Header.tsx

```diff
@@ -4,52 +4,73 @@ import Link from 'next/link';
 import { useRouter } from 'next/navigation';
 
 import WineLogoIcon from '@/app/assets/icons/wine-logo';
+import useLogout from '@/hooks/useLogout';
+import useUserStore from '@/stores/Auth-store/authStore';
+
+import Dropdown from './Dropdown';
+import ProfileImg from './ProfileImg';
 
 export default function Header() {
-  // const [isLogin, setIsLogin] = useState(false); //추후에 zustand store에서 state를 가져와 User정보 유무에 따라 조건부렌더링으로 수정
   const router = useRouter();
+  const user = useUserStore((state) => state.user);
+
+  const logout = useLogout();
+  const setUser = useUserStore((state) => state.setUser);
+
+  const handleLogout = async () => {
+    try {
+      await logout();
+      setUser(null);
+    } catch (error) {
+      alert('로그아웃 실패');
+      console.error('로그아웃 실패:', error);
+    }
+  };
```

Comment:
로그아웃이 어짜피 쿠키만 삭제하는 작업이라면 굳이 비동기 처리를 할 필요는 없어보입니다.
어짜피 로그아웃 기능은 헤더에서만 사용할 기능이므로 useLogout훅을 사용할 필요 없이 그냥 여기서 바로 구현하는 게 나아보이는 데 어떠신가요? 
```suggestion
  const router = useRouter();
  const clearUser = useUserStore((state) => state.clearUser);
  
  const handleLogout = () => {
    document.cookie = 'accessToken=; path=/; max-age=0';
    document.cookie = 'refreshToken=; path=/; max-age=0; httpOnly';
    clearUser();
    router.push('/login');
  };
```